src/domain/accounts-ips/ip-metadata-validator.ts

```diff
@@ -11,7 +11,7 @@ export const IPMetadataValidator = ({
   denyASNs,
   allowASNs,
 }: IPMetadataValidatorArgs): IPMetadataValidator => {
-  const validateForReward = (ipMetadata?: IPType): true | ValidationError => {
+  const validate = (ipMetadata?: IPType): true | ValidationError => {
```

Comment:
I think we are conflating 2 issues here. Validation should only be concerned with the shape and bounds of data - does the input conform to our business rules ie. can it be represented as a domain type.

Wether or not to allow a login given a (valid) IP address is not validation. Is a different flavour of concern.

Reply:
are you saying the word `validate` is the issue here? because it's been used differently in other context in our app?